src/App.js

```diff
@@ -1,20 +1,67 @@
-import React from "react";
-import logo from "./logo.png";
+import React, { useState } from "react";
 import "./App.css";
+import axios from "axios";
 
-class App extends React.Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-        </header>
+function App() {
+  const [input, setInput] = useState("");
+  const [data, setData] = useState({});
+  const [cityName, setCityName] = useState("");
+  const API_KEY = "7c52b0b346e3515a9d1950f5fd508f09";
```

Comment:
never hardcode your api keys into your code. This is a big security risk and can lead to you paying lots of dollars by someone else misusing this. Since this github repo is publicly available, you could face big issues with this if it was a google api key or something else that charges by number of requests. 

Please revise https://bootcamp.rocketacademy.co/2-full-stack/2.2-advanced-react/2.2.7-environmental-variables